mbof/views.py

```diff
@@ -123,9 +149,37 @@ class IntentionViewSet(OcellusViewSet):
     queryset = Intention.objects.all()
     serializer_class = IntentionSerializer
 
+    def retrieve(self, request, *args, **kwargs):
+        log_action(request)
+        instance = self.get_object()
+        serializer = self.get_serializer(instance)
+        serializer_data = serializer.data
+        if serializer_data['respondent'] == self.request.user.username:
```

Comment:
Use Boolean rather than string, too.  And reconsider whether this logic is needed at all.  Let the code that uses this data make the comparison.